webapp-django/crashstats/authentication/tests/test_migration.py

```diff
@@ -1,103 +0,0 @@
-import contextlib
```

Comment:
We don't need this anymore--it's for migration users from their email alias to their LDAP account.